### PR TITLE
Linked ideas 

### LINKED ISSUE #2: Create more structured content for intermediate ideas
1. Create `_ideas` folder in root.
1. Create content for:
    
    **Intermediate Heatmap**
    * Create a dropdown that allows users to select what course they want to see the heatmap for.
    * Add a legend to the heatmap showing how many posts each shade of blue represents.
    * When a cell in the heatmap is hovered, display a count of the number of posts.
    * Create a date range selector that will filter the data so that the heatmap displays only the data that fits within the specified date range.
    * Aggregate discussion from all of your courses instead of one course.
    * Create a way to show switch between showing just TA/Instructor posts and all posts.
    
    **Advanced Heatmap**
    * When a cell in the heatmap is hovered or clicked, show the discussions that were posted at that date and time.
    * Create a calendar view so that it is possible to navigate between weeks/months.
    * Create a filter that allows users to select a specific individual to see when they post.
    * Run text/sentiment analysis on the discussion posts and use this data to create a heatmap.

    **Intermediate New Ideas**
    * Create a degree planning tool that shows students what courses they are eligible to register for, given a list of courses in Canvas students have completed and the [degree requirments data (CSV)](TODO: LINK)
    * More ideas?

    **Advanced New Ideas**
    * ??

Intermediate may have code snippets, advanced will have none, just text.

If this is too much work, we can cut down on the suggestions, or just write up steps for a subset of them.

### LINKED ISSUE #59: Once _ideas are complete, remove numbering
The numbers in the names of the individual ideas don't make any sense, remove them once we link them back to https://github.com/UBC-LA-Hackathon/student-dashboard/blob/master/6-Create-Advanced-Data-Viz.md

### PR DESCRIPTION
**Pull request by justin0022**
We won't have all of the tasks written, so I think we should merge what we do have into master. If we end up writing more we can merge them in.

### CODE REVIEW

**Comment by justin0022** [comment]
I've also created stubs for the tasks that have not been written - maybe some hackathon participants can contribute and add in the steps.

File: _ideas/heatmap/intermediate/aggregate-discussions.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+## Heatmap - Intermediate
+
+## Task
+Aggregate discussion from all of your courses instead of one course.
+
+## Overview
+Right now you've hard-coded your Canvas course id in [server](../../../backend/server.js) in the call to `getDiscussions`. Instead of hard-coding this course id, we'll use [`getCoursesByUser`](https://github.com/ubccapico/node-canvas-api/blob/master/src/getCoursesByUser.js) to retrieve all of the Canvas courses you have access to. `getCoursesByUser` takes as input a Canvas user id, which you can hard-code or be extra fancy about and call `getSelf` to retrieve (this is up to you).
```
Comment (markoprodanovic) [inline _ideas/heatmap/intermediate/aggregate-discussions.md]: No longer hard-coded Canvas course id -- in [5-Create Course-Dropdown.md](https://github.com/UBC-LA-Hackathon/student-dashboard/blob/master/5-Create-Course-Dropdown.md) we now already suggest students use`getCoursesByUser` to populate dropdown with course names.

**Review by markoprodanovic — COMMENTED** [review]
Looks good overall! See comments made for `aggregate-discussions.md` - which goes over changes to `getDiscussions` that have already been discussed in [5-Create Course-Dropdown.md](https://github.com/UBC-LA-Hackathon/student-dashboard/blob/master/5-Create-Course-Dropdown.md)

File: _ideas/heatmap/intermediate/aggregate-discussions.md (hunk 1 above)
Comment (justin0022) [reply _ideas/heatmap/intermediate/aggregate-discussions.md]: Good catch, I removed the reference

**Review by markoprodanovic — APPROVED** [review]
👍

**Pull request merged**